server/services/RunService.js

```diff
@@ -631,20 +631,10 @@ async function getActiveRunsConfiguration() {
  * @return {Array.<TestSuiteVersion>}
  *
  */
-async function getNewTestVersions() {
+async function getTestVersions() {
     try {
-        const activeTestVersion = await db.TestVersion.findOne({
-            where: { active: true },
-            attributes: ['datetime']
-        });
-        let where = {};
-        if (activeTestVersion) {
-            where = {
-                datetime: { [db.Sequelize.Op.gte]: activeTestVersion.datetime }
-            };
-        }
         return await db.TestVersion.findAll({
-            where: where,
+            where: {},
```

Comment:
Curious as to why the filtering was removed here and moved to the frontend?

Reply:
Because we need the old tests too in order to show historical versions on the test reports pages.